molecule/default/tests/test_reboot_devices.py

```diff
@@ -0,0 +1,63 @@
+import os
+import re
+import pytest
+import testinfra.utils.ansible_runner
+
+# TODO: will add a filter
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('utility_container')[:1]
+
+pre_cmd = "sudo bash -c \"source /root/openrc; "
+
+
+@pytest.mark.jira('asc-228')
+@pytest.mark.skip(reason='add skipped tests for asc-301')
+def test_to_reboot_nova_compute(host):
+    cmd = pre_cmd + "ansible -m shell nova_compute -a \'reboot\' \""
+    assert host.run_expect([0], cmd)
+    verify_the_device_is_backup(host, 'test_cmd')
```

Comment:
I would prefer:
1. That there is some implementation of the `verify_the_device_is_backup()` method
2. That the method be renamed to `verify_the_device_is_back_up` to remove any confusion about what the method does ("backup" vs "back up"). I miss read this as "is it a backup".
3. That a "real" `test_cmd` be put here even if it is something like `true`.